test/api_fixtures/canvas_subs.json

```diff
@@ -1,22 +1,36 @@
-[
-    {
-        "id": 444444,
-        "score": 125.0,
-        "submitted_at": "2020-06-19T17:45:33Z",
-        "assignment_id": 111112,
-        "graded_at": "2020-06-19T17:45:33Z",
-        "user": {
-            "login_id": "hpotter"
+{
+    "Potions_Validation" : [
+        {
+            "id": 444444,
+            "score": 125.0,
+            "submitted_at": "2020-06-19T17:30:05Z",
```

Comment:
As you can see above, this value was originally the same as `graded_at`. I modified them and a test in `test_orch` so they were different.

Reply:
Have you seen `graded_at` and `submitted_at` date timestamp are different when you tested with real courses?

Reply:
Yes, with Spanish, sometimes they are the same, sometimes they are a couples seconds apart. If the exam was graded in some manual way, they would definitely be different.